.vscode/settings.json

```
@@ -14,7 +14,7 @@
   },
   "typescript.tsdk": "./node_modules/typescript/lib",
   "editor.codeActionsOnSave": {
-    "source.fixAll": true
+    "source.fixAll": "explicit"
```

This change is unrelated to the PR. Suggesting, revert.

```suggestion
    "source.fixAll": true
```